README.md

```diff
@@ -1,2 +1,36 @@
 # squash-apps
 A place to develop new visualizations or dashboards that you would like to deploy to SQuaSH
+
+
+1. Clone this repo
+
+```
+git clone https://github.com/lsst-sqre/squash-apps.git
+```
+
+2. Create a virtualenv
+
+```
+cd squash-apps
+
+virtualenv squash-apps -p python3
```

Comment:
I know embarrassingly little about `virtualenv`s.  Why are you suggesting one in this case?

Reply:
I could use `conda env` or similar it's just a way to start a clean python environment and don't mess around with packages installed globally. You can find a good description here http://docs.python-guide.org/en/latest/dev/virtualenvs/ including wrappers around it.